src/line_absorption.jl

```diff
@@ -212,6 +210,8 @@ function hydrogen_line_absorption!(αs, λs, T, nₑ, nH_I, UH_I, ξ, window_siz
         ν₀ = c_cgs / (λ₀)
         scaled_Δν = _zero2epsilon.(abs.(view(νs,lb:ub) .- ν₀) ./ F0)
         dIdν = exp.(line.profile.(T, nₑ, log.(scaled_Δν)))
+        # interpolating between -Infs can lead to NaNs which should be 0
```

Comment:
Isn't it better to clip the -inf to a -BIG_NUMBER instead ? 
Masking nans is indeed a bit worrysome.